docs/docs/sources/facebook.md

```diff
@@ -99,4 +120,120 @@ On the `User or Page` option, select `Get Page Token` and click on `Generate Acc
 
 <img alt="Facebook token page" src={useBaseUrl('img/sources/facebook/token.jpg')} />
 
-You're now ready to connect a Facebook page to your Airy Core instace 🎉.
+<SuccessBox>
+
+You are now ready to connect a Facebook page to your Airy Core instance 🎉
+
+</SuccessBox>
+
+<br />
+
+## Connect a Facebook page to your instance
+
+The next step is to send a request to the [Channels endpoint](/api/endpoints/channels#facebook) to connect a Facebook page to your instance.
+
+<ButtonBox
+icon={() => <BoltSVG />}
+title='Channels endpoint'
+description='Connect a Facebook source to your Airy Core instance through the Channels endpoint'
+link='api/endpoints/channels#facebook'
+/>
+
+<br />
+
+```
+POST /channels.facebook.connect
```

Comment:
If we want to duplicate the endpoint documentation here, I would add them as a `connect-facebook.mdx` component so that we make it easier to change in the future.

Reply:
Agreed.

To be fair we moved this away from the sources document on the suggestion of @steffh but I suppose then @sgarroum suggested we introduced this back?

I think what @chrismatix suggested is the best of both words as, in fact, both places are "right" for this kind of doc. 

@AudreyKj we do something along the line of this with twilio config so you can use that as a reference (do ping me if you run into any trouble)

Reply:
Yes, I am highly in favor of duplicating it (@chrismatix unification approach makes sense) and thus making the source connection doc self sufficient - from the first connection command to testing it by sending and receiving a message.